src/assignments/2-operator.js

```diff
@@ -13,6 +13,24 @@ output example
 20%10=0
 */
 // start here writing the code for program1
+/* eslint-disable */ 
+const num1 = 20
+const num2 = 10
+
+const add = num1 + num2
+console.log(add)
+
+const subtract = num1 - num2
+console.log(subtract)
+
+const multiply = num1 * num2
+console.log(multiply)
+
+const remanider = num1 / num2
+console.log(remanider)
+
+const modulo = num1 % num2
```

Comment:
if is better to check whether the num2 is bigger than 0, then, do the calculation